app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parsed_command = command.split()
```

Comment:
The code style issue. According to the checklist, descriptive variable names should be used. The variable name 'parsed_command' could be more descriptive. Consider renaming it to something more meaningful, such as 'command_parts' or 'command_components'.